Makefile

```diff
@@ -39,6 +39,10 @@ kind-deploy:
 	@echo "Wait for pods with: kubectl get pods,svc"
 	@echo "See k8s/README.md for more info."
 
+kind-create-schema-configmap:
```

Comment:
I assume this for explicitly loading a yaml, when we want it to be different from the one in bootstrap? Because the container would start with the bootstrap file

Reply:
ignore my comment.